src/main/drivers/pwm_output.c

```diff
@@ -460,8 +456,8 @@ void servoDevInit(const servoDevConfig_t *servoConfig)
             /* flag failure and disable ability to arm */
             break;
         }
-
-        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_MHZ, 1000000 / servoConfig->servoPwmRate, servoConfig->servoCenterPulse, 0);
+        const uint16_t period = PWM_TIMER_HZ / servoConfig->servoPwmRate;
+        pwmOutConfig(&servos[servoIndex], timer, PWM_TIMER_HZ, period, (servoConfig->servoCenterPulse - 1000) / 1000 * period, 0);
```

Comment:
Original code is stylistically incorrect - it combines PWM_TIMER_HZ (it is reasonable to assume that this constant can be changed) and hardcoded values.

Renaming PWM_TIMER_HZ to PWM_TIMER_1MHZ or similar may help ... 

The idea wat to fix the code to work independently of PWM_TIMER_HZ (within reasonable bounds), but this can wait for another PR